src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskSearchControllerTest.java

```diff
@@ -168,12 +168,14 @@ public void should_return_a_200_with_search_results_and_correct_properties_for_g
         List<String> taskIds = tasksCreated.stream().map(TestVariables::getTaskId).collect(Collectors.toList());
         List<String> caseIds = tasksCreated.stream().map(TestVariables::getCaseId).collect(Collectors.toList());
 
-        SearchTaskRequest searchTaskRequest = new SearchTaskRequest(asList(
-            new SearchParameterList(JURISDICTION, SearchOperator.IN, singletonList("WA")),
-            new SearchParameterList(LOCATION, SearchOperator.IN, singletonList("765324")),
-            new SearchParameterRequestContext(REQUEST_CONTEXT, SearchOperator.CONTEXT, RequestContext.ALL_WORK),
-            new SearchParameterList(CASE_ID, SearchOperator.IN, caseIds)
-        ));
+        SearchTaskRequest searchTaskRequest = new SearchTaskRequest(
+            RequestContext.ALL_WORK,
```

Comment:
Do we also have a test (maybe existing) where the `request_context` is absent in the request?  This is a valid scenario since it is an optional field.

Reply:
Yes, there other tests sending without the request_context